multicluster/apis/multicluster/v1alpha1/multiclusterconfig_types.go

```diff
@@ -53,6 +53,10 @@ type MultiClusterConfig struct {
 	// PodIP type requires Multi-cluster Gateway too when there is no direct Pod-to-Pod
 	// connectivity across member clusters.
 	EndpointIPType string `json:"endpointIPType,omitempty"`
+	// Enable StretchedNetworkPolicy which will export and import labelIdentities in the
+	// ClusterSet and allow Antrea-native policies to select peers from other clusters
+	// in a ClusterSet.
+	EnableStretchedNetworkPolicy bool `json:"enableStretchedNetworkPolicy,omitempty"`
```

Comment:
Do we add the toggle to leave the control of enabling this feature to user for long term or use it to isolate experimental code?

Reply:
I believe the former. We want to eliminate label identity exchange / having traffic sent out by Pods carry the label identity info in their registers, if the toggle is turned off